brew_view/__init__.py

```diff
@@ -40,140 +38,138 @@
 def setup_brew_view(spec, cli_args):
     global config, logger, app_log_config, event_publishers, notification_meta
 
-    # We load the config once just to see if there is a config file we should load
-    temp_config = spec.load_config(cli_args, 'ENVIRONMENT')
+    config_sources = [cli_args, 'ENVIRONMENT']
```

Comment:
This code is basically the same as `bartender/__init__.py` We should consider moving it to `bg_utils`

Reply:
:+1: